Makefile

```diff
@@ -64,3 +64,13 @@ clean:
 	"$(GO)" clean -cache -testcache -r
 	@echo "removing generated files"
 	@rm -rf build
+
+.PHONY: go-lint
+go-lint: ## run go linters.
+	@echo '>>> Running go linters.'
+	@golangci-lint run -v --issues-exit-code 0 ## TODO: remove after fixing all lint issues
+
+.PHONY: install-tools
+install-tools: ## install tools.
+	@echo '>>> Installing tools.'
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.54.2
```

Comment:
I have been using `v1.58.0` for a while, there might be an even newer version - can this be updated?

Reply:
i am testing this, and will get back to you, as i have some issues

Reply:
bumped to latest one, please review again